src/test/integration/controllers/user.spec.js

```diff
@@ -66,7 +66,7 @@ describe('User controller', () => {
   let app, server
 
   beforeAll(async () => {
-    ({ app, server } = await serverInit())
+    ;({ app, server } = await serverInit())
```

Comment:
Could you please explain why there is a semicolon before this line?

Reply:
According to our Prettier project settings, we don't put the semicolon at the end of a statement. So, to ensure that the statement doesn't accidentally concatenate with the previous statement, the semicolon was placed there by Prettier.
[Here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#automatic_semicolon_insertion) you can find more info with examples.

Reply:
Thanks!